src/di/BindValidator.php

```
@@ -42,7 +42,7 @@ public function to(string $interface, string $class): ReflectionClass
             throw new InvalidType("[{$class}] is no implemented [{$interface}] interface");
         }
 
-        return new ReflectionClass($class); // @phpstan-ignore-line
+        return new ReflectionClass($class);
```

PHPStan (2.0.3)

Method Ray\Di\BindValidator::to() should return Ray\Aop\ReflectionClass<T of object> but returns Ray\Aop\ReflectionClass<object>.


(return.type)